### ISSUE TITLE
Verify Spanish locale files are updated

### ISSUE THREAD

**Issue by tlongers** [security-force-monitor/sfm-cms#685]
@jeancochrane We are also updating the Spanish translation. Can you verify that the `es` .po file is updated to reflect the updated `~/about` page text before we do so?

**Comment by jeancochrane** [comment]
We run `makemessages` on every deploy so I'm pretty sure we're good, but I just ran it manually on the staging server to make extra sure 👍

**Comment by tlongers** [comment]
It looks okay, but the string count is different from the French locale - is this how it should be?

<img width="656" alt="image" src="https://user-images.githubusercontent.com/2115933/82345619-62ac9d80-99ed-11ea-9844-c54081e55bb6.png">

**Comment by jeancochrane** [comment]
I'm pretty sure that's right. The way that Rosetta counts messages is confusing when there's an existing .po file with outdated translations, and unfortunately I still don't have a principled way of making sure the math adds up. But I checked against the production Rosetta instance and there's a substantial difference between the two Spanish message counts, which is a good signal; I also double-checked the last tag that I updated, which was changing the text on the downloads page to say `Download information about units` instead of `Download information about organizations`, and the change is reflected on the staging instance but not yet on the production instance.
